Source/WTF/wtf/GenerateProfiles.h

```diff
@@ -62,7 +62,7 @@ ALWAYS_INLINE void registerProfileGenerationCallback(const char* name)
         {
             // Maybe we could use %t instead here, but this folder is permitted through the sandbox because of ANGLE.
             FileSystem::PlatformFileHandle fileHandle;
-            auto filePath = FileSystem::openTemporaryFile(makeString(name, "-", getpid()), fileHandle, ".profraw");
+            auto filePath = FileSystem::openTemporaryFile(makeString(name, "-", getpid()), fileHandle, makeString(".profraw"));
```

Comment:
I'm out of the loop here, do we use `makeString` or `".profraw"_s` these days?

Reply:
This is unnecessarily inefficient, this should be ".profraw"_s

Reply:
The proposed change would construct an unnecessary temporary String()